cmd/object-handlers.go

```diff
@@ -88,6 +89,41 @@ func (f funcToWriter) Write(p []byte) (int, error) {
 	return f(p)
```

Comment:
We can get rid of the above code if not used any more @aead

Reply:
Yes we should..if this is better.

Reply:
okay